packages/flutter/test/material/text_selection_test.dart

```diff
@@ -547,6 +547,79 @@ void main() {
+        final TextEditingController controller =
+            TextEditingController(text: 'abc\ndef\nghi\njkl\nmno\npqr');
+        await tester.pumpWidget(MaterialApp(
+          theme: ThemeData(platform: TargetPlatform.android),
+          home: Directionality(
+            textDirection: TextDirection.ltr,
+            child: MediaQuery(
+              data: const MediaQueryData(size: Size(800.0, 600.0)),
+              child: Align(
+                alignment: Alignment.bottomCenter,
+                child: Material(
+                  child: TextField(
+                    style: const TextStyle(fontSize: 32, height: 1),
+                    maxLines: 2,
+                    controller: controller,
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ));
+
+        // Initially, the menu isn't shown at all.
+        expect(find.text('Cut'), findsNothing);
+        expect(find.text('Copy'), findsNothing);
+        expect(find.text('Paste'), findsNothing);
+        expect(find.text('Select all'), findsNothing);
+        expect(find.byType(IconButton), findsNothing);
+
+        // Tap to place the cursor in the field, then tap the handle to show the
+        // selection menu.
+        await tester.tap(find.byType(TextField));
+        await tester.pumpAndSettle();
+        final RenderEditable renderEditable = findRenderEditable(tester);
+        final List<TextSelectionPoint> endpoints = globalize(
+          renderEditable.getEndpointsForSelection(controller.selection),
+          renderEditable,
+        );
+        expect(endpoints.length, 1);
+        final Offset handlePos = endpoints[0].point + const Offset(0.0, 1.0);
+        await tester.tapAt(handlePos, pointer: 7);
+        await tester.pumpAndSettle();
+        expect(find.text('Cut'), findsNothing);
+        expect(find.text('Copy'), findsNothing);
+        expect(find.text('Paste'), findsOneWidget);
+        expect(find.text('Select all'), findsOneWidget);
+        expect(find.byType(IconButton), findsNothing);
+
+        // Tap to select all.
+        await tester.tap(find.text('Select all'));
+        await tester.pumpAndSettle();
+
+        // Only Cut, Copy, and Paste are shown.
+        expect(find.text('Cut'), findsOneWidget);
+        expect(find.text('Copy'), findsOneWidget);
+        expect(find.text('Paste'), findsOneWidget);
+        expect(find.text('Select all'), findsNothing);
+        expect(find.byType(IconButton), findsNothing);
+
+        final Offset cutOffset = tester.getTopLeft(find.text('Cut'));
+        final Offset textFieldOffset =
+            tester.getTopLeft(find.byType(TextField));
+        expect(cutOffset.dy + 25, equals(textFieldOffset.dy));
+      },
+      skip:
+          isBrowser, // [intended] We do not use Flutter-rendered context menu on the Web.
+      variant: const TargetPlatformVariant(
+          <TargetPlatform>{TargetPlatform.android, TargetPlatform.iOS}),
```

Comment:
just keep it one line

Reply:
and you need spaces in after `{` and before `}`

Reply:
fixed